### DISCUSSION TITLE
Fitur diskusi staff

### DISCUSSION THREAD

**Discussion by maxzycon** [upkoding/upkoding#9] [Ideas]
Selamat sore,
Ijin beri saran, ketika mau lihat proyek yg sedang direview terlalu banyak redirect, 

Cara pertama :
1). click nama proyek (misal Kalkulator sederhana)
2). click lihat 

Cara kedua :
1). search proyek
2). click lihat

baru bisa menuju diskusi para reviewer, yang mana menurut saya kurang efektif 

saran saya agar dibuatkan 1 menu yg didalamnya ada list2 dari proyek yg sedang / sudah direview oleh staff 
jadi tinggal klik menu itu saja 

terima kasih

**Comment by ekaputra07** [comment]
Terima kasih @nextindev idenya menarik 👍

Iya betul sekali seperti itu, karena memang diskusi staff hanya bisa dilihat dari dalam proyek yg dikerjakan usernya sendiri. Jadi memang perlu banyak klik hehe. Soalnya saya tidak mengira ada yg tertarik dengan diskusi yang terjadi di proyek lain.

Thanks ya nanti coba saya liat bagaimana cara terbaik melakukkannya.

Ide2 dan masukan seperti ini yang ingin saya cari sebanyak2nya.  Keep'em coming pelase! 🙏

**Comment by ekaputra07** [comment]
@devnazir bagaimana menurut anda tentang yang ini? hehe

**Reply by devnazir** [reply]
@ekaputra07 iya bli, memang terlalu banyak redirect sih. buat menu review itu kayaknya bakal lebih efektif. Tpi, fitur sblmnya juga jgn dihilangkan karena itu bisa jadi cara kedua untuk melihat review.

**Reply by ekaputra07** [reply]
Iya betul, yang saat ini akan tetap disana, cuma perlu ditambahkan halaman untuk mengakses/memantau semua review dengan mudah.
